conans/test/unittests/model/build_info/components_test.py

```diff
@@ -243,15 +243,15 @@ def _assert_link_order(sorted_libs):
         self.assertEqual(["libK", "libJ", "libG", "libH", "libL", "libF", "libI", "libC", "libD",
                           "libE", "libA", "libB"], dep_cpp_info.libs)
         deps_cpp_info.add("dep2", dep_cpp_info)
-        self.assertEqual(["lib6", "lib5", "lib4", "lib3", "lib1", "lib2","libK", "libJ", "libG",
+        self.assertEqual(["lib6", "lib5", "lib4", "lib3", "lib1", "lib2", "libK", "libJ", "libG",
                           "libH", "libL", "libF", "libI", "libC", "libD", "libE", "libA", "libB"],
                          list(deps_cpp_info.libs))
 
     def cppinfo_inexistent_component_dep_test(self):
         info = CppInfo("", None)
         info.components["LIB1"].requires = ["LIB2"]
         with six.assertRaisesRegex(self, ConanException, "Component 'LIB1' "
-                                                         "declares a missing dependency"):
+                                                         "missing dependencies: 'LIB2'"):
```

Comment:
I think we need to improve the message, but we can make it more explicit: _"Component 'LIB1' requires component 'LIB2' but it is not found in this package"_